Go.gitignore

```diff
@@ -11,6 +11,9 @@
 # Test binary, built with `go test -c`
 *.test
 
+# Default Delve debug binary
+__debug_bin
```

Comment:
```suggestion
__debug_bin*
```

There's a new change to delve to include timestamp in output
source: https://github.com/go-delve/delve/commit/801a9109c7fd7be8b060b466cc041a3695455c77